### ISSUE TITLE
Spectrogram NaN value while training

### ISSUE THREAD

**Issue by imiles2021** [DigitalPhonetics/IMS-Toucan#214]
Hi Florian,

Thanks again for your work, and I know you must be very busy!

But I'm just wondering if you had any other thoughts on this issue about the spectrogram values turning to NaN after the first training iteration. I've been struggling with this for a while and haven't managed to find a solution.

[https://github.com/DigitalPhonetics/IMS-Toucan/issues/73]

I've ran the scorer to get rid of high datapoints / those that turn to NaN. 

Thanks,
Isobelle Miles

**Comment by Flux9665** [comment]
Hi Isobelle! I need to know a couple more details to narrow it down:

- How many datapoints do you have in the train set? 
- How many did the scorer remove? 
- What's your learning rate? 
- What's your batchsize?

Usually I would expect this to happen only if there are datapoints where the text and the audio have a mismatch, but the scorer should have taken care of them already.

**Comment by imiles2021** [comment]
Hi again,

Thanks for your quick reply!!

Actually I think I had a problem with my code and I wasn't running the scorer as I expected - the training is running smoothly now.

Thanks again and sorry for the misunderstanding!

**Comment by Flux9665** [comment]
happy to hear it works now! Let me know if there are other issues :) With low-resource languages there's usually a new problem at every corner
